Bug - cannot find module babel-eslint in v0.14.22 and v0.14.23

> Cannot find module 'babel-eslint'
Referenced from: /Users/jenweber/cardstack/project-template/cardhost/.eslintrc.js

It seems [this refactoring](https://github.com/cardstack/cardstack/commit/109b7b988390ad6390cf5f0c61d3435635d324a2#diff-b0b9be13694993c7d72b14c28d8226e8) dropped inheriting from node, which it appears was important for existing apps. Here's what the config looked like [before](https://github.com/cardstack/cardstack/blob/v0.14.21/packages/eslint-config/browser.js).

Here's an example of an app that uses these configs: [project-template](https://github.com/cardstack/project-template/find/master)

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -9,8 +9,6 @@ and features, and is not intended to be an exhaustive list.
 
 ## [v0.14.22] - 2019-08-15
 
-`yarn upgrade-interactive --exact 0.14.22 --scope @cardstack`
-
 When upgrading, `@cardstack/eslint-config/browser` should become `@cardstack/eslint-config/ember-addon`
 throughout a project. Then, `babel-eslint` should be added as a peer dependency in the top level 
 `package.json` with `yarn add -D -W babel-eslint`

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -9,6 +9,12 @@ and features, and is not intended to be an exhaustive list.
 
 ## [v0.14.22] - 2019-08-15
 
+`yarn upgrade-interactive --exact 0.14.22 --scope @cardstack`
+
+When upgrading, `@cardstack/eslint-config/browser` should become `@cardstack/eslint-config/ember-addon`
+throughout a project. Then, `babel-eslint` should be added as a peer dependency in the top level 
+`package.json` with `yarn add -D -W babel-eslint`
+
 ### Added
 
 - Commits for the WIP Cardstack CLI
@@ -19,8 +25,7 @@ and features, and is not intended to be an exhaustive list.
 - error logging
 - docs for git plugins
 - more linting configuration to support using TypeScript
-- eslint plugin rules have changed filepaths for imports. `@cardstack/eslint-config/browser` should become `@cardstack/eslint-config/ember-addon` throughout a project. Then, `babel-eslint` should be added as a peer
-dependency in the top level `package.json` with `yarn add -D -W babel-eslint` #824
+- Breaking - eslint plugin rules have changed filepaths for imports. See migration instructions above. #824
 
 ## [v0.14.21] - 2019-07-18
 

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -19,7 +19,8 @@ and features, and is not intended to be an exhaustive list.
 - error logging
 - docs for git plugins
 - more linting configuration to support using TypeScript
-- eslint plugin rules have changed filepaths for imports. `@cardstack/eslint-config/browser` should become `@cardstack/eslint-config/-browser`, with a dash before `browser` #824
+- eslint plugin rules have changed filepaths for imports. `@cardstack/eslint-config/browser` should become `@cardstack/eslint-config/ember-addon` throughout a project. Then, `babel-eslint` should be added as a peer
+dependency in the top level `package.json` with `yarn add -D -W babel-eslint` #824
 
 ## [v0.14.21] - 2019-07-18
 

